setup.cfg

```diff
@@ -1,6 +1,90 @@
-[flake8]
-application-import-names = mopidy_beets,tests
-exclude = .git,.tox
+[metadata]
+name = mopidy-beets
+version = 3.1.0
+url = https://github.com/mopidy/mopidy-beets
+author = Lars Kruse
+author_email = devel@sumpfralle.de
+license = MIT
+license_file = LICENSE
+description = Beets extension for Mopidy
+long_description = file: README.rst
+classifiers =
+    Environment :: No Input/Output (Daemon)
+    Intended Audience :: End Users/Desktop
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Multimedia :: Sound/Audio :: Players
+
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+python_requires = >= 3.7
+install_requires =
+    Mopidy >= 3.0.0a4  # Change to >= 3.0 once final is released
+    Pykka >= 2.0.1
+    requests >= 2.0.0
+    setuptools
+
+
+[options.extras_require]
+lint =
+    black
+    check-manifest
+    flake8
+    flake8-bugbear
+    flake8-import-order
+    isort[pyproject]
+release =
+    twine
+    wheel
+test =
+    pytest
+    pytest-cov
+dev =
+    %(lint)s
+    %(release)s
+    %(test)s
+
+
+[options.packages.find]
+exclude =
+    tests
+    tests.*
 
-[wheel]
+
+[options.entry_points]
+mopidy.ext =
+    beets = mopidy_beets:BeetsExtension
+
+
+[bdist_wheel]
 universal = 1
+
+
+[flake8]
+application-import-names = mopidy_{{ cookiecutter.ext_name }}, tests
+max-line-length = 80
+exclude = .git, .tox, build
+select =
+    # Regular flake8 rules
+    C, E, F, W
+    # flake8-bugbear rules
+    B
+    # B950: line too long (soft speed limit)
+    B950
+    # pep8-naming rules
+    N
+ignore =
+    # E203: whitespace before ':' (not PEP8 compliant)
+    E203
+    # E501: line too long (replaced by B950)
+    E501
+    # W503: line break before binary operator (not PEP8 compliant)
+    W503
+    # B305: .next() is not a thing on Python 3 (used by playback controller)
+    B305
```

Comment:
Mopidy-Beets probably doesn't need to ignore `B305`. That's mostly useful for frontends that are using the `.next()` method in the Core API.

Reply:
Removed, thanks!